scripts/generate-versions.sh

```diff
@@ -17,12 +44,13 @@ fi
 
 cat <<-EOF
 {
-  "alertmanager": "$(get_latest_version "prometheus/alertmanager")",
-  "blackboxExporter": "$(get_latest_version "prometheus/blackbox_exporter")",
-  "grafana": "$(get_latest_version "grafana/grafana")",
-  "nodeExporter": "$(get_latest_version "prometheus/node_exporter")",
-  "prometheus": "$(get_latest_version "prometheus/prometheus")",
-  "prometheusAdapter": "$(get_latest_version "kubernetes-sigs/prometheus-adapter")",
-  "prometheusOperator": "$(get_latest_version "prometheus-operator/prometheus-operator")"
+  "alertmanager": "$(get_version "prometheus/alertmanager")",
+  "blackboxExporter": "$(get_version "prometheus/blackbox_exporter")",
+  "grafana": "$(get_version "grafana/grafana")",
+  "kubeStateMetrics": "$(get_version "kubernetes/kube-state-metrics")",
```

Comment:
Unrelated: Do you think we should enable kube-state-metrics 2.0 now? cc @brancz wdyt? We have an rc.0, or should we wait until rc.1?

Reply:
This is a chicken and egg kind of problem. We would like to enable rc to have more users test it but at the same time kube-prometheus is advertised as a sort of rolling-release where we cut release branches only for k8s compatibility. Additionally, since we are using stable versions for all components and were never using rc I am a bit hesitant to break this pattern.